src/Components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Tabs.scss';
+
+type Props = {
+  tabChangeMethod: (tabId: Tab) => void,
+  tabs: Tab[],
+  selectedTabId: string,
+};
+
+export const Tabs:React.FC<Props> = ({
+  tabChangeMethod,
+  tabs,
+  selectedTabId,
+}) => {
+  const selectedObj = tabs.find(tab => tab.id === selectedTabId);
+
+  const classNameMetod = (currentTabId: string) => {
+    if (currentTabId === selectedTabId) {
+      return 'tabs__button active';
+    }
+
+    return 'tabs__button';
+  };
+
+  return (
+    <div className="tabs">
+      <div>
+        {tabs.map(tab => (
+          <button
+            key={tab.id}
+            className={classNameMetod(tab.id)}
```

Comment:
it will be better if you use library className

Reply:
You are right, thanks